crates/pixi_manifest/src/toml/build_backend.rs

```diff
@@ -19,6 +17,7 @@ pub struct TomlPackageBuild {
     pub backend: PixiSpanned<TomlBuildBackend>,
     pub channels: Option<PixiSpanned<Vec<NamedChannelOrUrl>>>,
     pub additional_dependencies: UniquePackageMap,
+    pub src: Option<SourceSpec>,
```

Comment:
I would feel better if we change this type. You are currently using the `SourceSpec` type which is used to identify a source package. In the future we can add fields to this like a version or build string match that make no sense for the src field. My rationale for this is that a SourceSpec should ideally consistent of a SourceLocationSpec (e.g. git, path, etc.) and more information like `build_string` (because a single source location can output multiple packages).

We can also do this refactor when we add things to SourceSpec but Im a bit hesitant because it will be very easy to overlook doing this refactor.

Ill leave this up to you!